spring-cloud-dataflow-admin-local/src/main/java/org/springframework/cloud/dataflow/admin/spi/local/LocalConfiguration.java

```diff
@@ -25,21 +27,42 @@
 
 /**
  * Creates deployers that will deploy modules in-process.
- *
  * @author Eric Bottard
  */
 @Configuration
-@Import(ModuleLauncherConfiguration.class)
 public class LocalConfiguration {
 
-	@Bean
-	public ModuleDeployer processModuleDeployer(ModuleLauncher moduleLauncher) {
-		return new LocalModuleDeployer(moduleLauncher);
+	static final String LOCAL_DEPLOYER_PREFIX = "deployer.local";
+
+	@ConditionalOnProperty(prefix = LocalConfiguration.LOCAL_DEPLOYER_PREFIX, name = "out-of-process", havingValue = "false", matchIfMissing = true)
```

Comment:
I think that the default should be out-of-process, and allow in-proces to be used as an option for debugging, etc.